spec/features/merchants/dashboard_spec.rb

```diff
@@ -144,4 +144,32 @@
       expect(page).to have_content(@merchant_1.top_5_customers[4].transaction_count)
     end
   end
-end
+
+  describe 'in items ready to ship section' do 
+    scenario 'i see a list of item names that have not been shipped' do 
+      expect(page).to have_content("Items Ready to Ship")
+
+      expect(page).to have_content(@item_1.name)
+      expect(page).to have_content(@item_2.name)
+      expect(page).to have_content(@item_3.name)
+      expect(page).to have_content(@item_4.name)
+      expect(page).to have_content(@item_5.name)
+    end
+
+    scenario 'i see the invoice id next to the item name' do 
+save_and_open_page
```

Comment:
Should remove this save and open page (I am pretty sure you can do this from GitHub. If you click the three dot menu in the top right of the changed file view there is an option to `edit file`. It makes it so that you don't have to pull down, edit, push up, open new PR, etc. for a small change like that.